zebrad/Cargo.toml

```
@@ -15,6 +15,7 @@ default-run = "zebrad"
 [features]
 # In release builds, don't compile debug logging code, to improve performance.
 default = ["release_max_level_info"]
+getblocktemplate-rpcs = ["zebra-rpc/getblocktemplate-rpcs"]
```

Let's activate the state feature as well?

```suggestion
getblocktemplate-rpcs = ["zebra-rpc/getblocktemplate-rpcs", "zebra-state/getblocktemplate-rpcs"]
```